### PR TITLE
CI: don't use all-features

### PR DESCRIPTION
**Pull request by neonphog**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,6 +3,7 @@
 # the gha workflow sets this globally
 # set this also for local executions so we get the same results
 F=RUSTFLAGS="-Dwarnings"
+A=slow_tests,glacial_tests,build_wasms,sqlite-encrypted,chc
```
Comment (maackle) [inline Makefile:6]: Previously we didn't include glacial tests (that's what differentiates glacial from slow, they're really too slow to run regularly). Does that seem OK?

```suggestion
A=slow_tests,build_wasms,sqlite-encrypted,chc
```

Comment (neonphog) [reply Makefile:6]: I guess I'd counter with, if we never run them is there any point in having them? Have the builds felt too slow with --all-features enabled? I don't have a strong opinion either way... happy to remove `glacial_tests` if that is what we want to do : )

Comment (neonphog) [reply Makefile:6]: @maackle - That works for me. We still don't have any process for running these glacial tests ever, but I suppose that doesn't have to be solved for this PR : )

Comment (maackle) [reply Makefile]: Done

**Comment by neonphog** [comment]
Since I opened this, we'll need your approval @maackle : )

**Review by maackle — APPROVED** [review]

**Pull request merged**